Common/Transforms/itkAdvancedVersorRigid3DTransform.hxx

```diff
@@ -46,8 +46,7 @@ AdvancedVersorRigid3DTransform<TScalarType>::AdvancedVersorRigid3DTransform()
 
 // Constructor with arguments
 template <class TScalarType>
-AdvancedVersorRigid3DTransform<TScalarType>::AdvancedVersorRigid3DTransform(unsigned int outputSpaceDim,
-                                                                            unsigned int paramDim)
+AdvancedVersorRigid3DTransform<TScalarType>::AdvancedVersorRigid3DTransform(unsigned int paramDim)
   : Superclass(paramDim)
```

Comment:
For the record, the initial version of this constructor passed `outputSpaceDim` to its own Superclass:

https://github.com/SuperElastix/elastix/blob/5d8252f2c3f3295855afcb6b88b147fdca665baa/src/Components/Transforms/SimilarityTransform/itkAdvancedVersorRigid3DTransform.txx#L36-L40